docs/bare_metal/bare_metal_api.md

```diff
@@ -1,5 +1,33 @@
 # Bare metal APIs
 
+The bare metal profile starts with a restricted list of API, to which you can add what you need. However, bare metal doesn't support all Mbed OS APIs.
```

Comment:
I am assuming that this page will be deleted.

Reply:
Yes.